packages/jaeger-ui/src/utils/date.tsx

```diff
@@ -27,7 +27,8 @@ const YESTERDAY = 'Yesterday';
 export const STANDARD_DATE_FORMAT = 'YYYY-MM-DD';
 export const STANDARD_TIME_FORMAT = 'HH:mm';
 export const STANDARD_DATETIME_FORMAT = 'MMMM D YYYY, HH:mm:ss.SSS';
-export const ONE_MILLISECOND = 1000;
+export const ONE_MICROSECOND = 1;
+export const ONE_MILLISECOND = 1000 * ONE_MICROSECOND;
```

Comment:
tbh, I don't find these constants intuitive at all. In contrast, in Go stdlib there are similar constants which have a clear semantic meaning because Go counts all times relative to 1ns granularity, so `const Millisecond=1_000_000` makes sense. Here I don't know what these constants mean, because the baseline is only implied. E.g.:
```
ScatterPlot.jsx:21:import { ONE_MILLISECOND } from '../../../utils/date';
```
wut?

Reply:
Separately, ONE_MICROSECOND is not used anywhere in the codebase, so why export it?

Reply:
The base unit for the time in jaeger-ui is microsecond, those constants represent the time unit expressed in microsecond

ONE_MICROSECOND = 1 microsecond
ONE_MILLISECOND = 1000 microsecond 
ONE_SECOND = 1000_000 microsecond and so on

I've exported the ONE_MICROSECOND constant just in case we need it in the future, using `ONE_MICROSECOND` is more readable and concise than just writing `1` as a literal

Reply:
ONE_MICROSECOND is not more readable, because it's meaningless without context. The context being "this is the value that represents 1ms as the amount of _default units_", where default units are not specified and instead implied. For example, say you see a code using a constant ONE_APPLE - what does it tell you? It could be weight in grams, it could be amount of calories - we don't know. It's the same story with these constants.

But even if they were meaningful, that's no justification for adding code and exporting constants that are not elsewhere used.  If/when they are used, we can add them.

Reply:
Oh got it, should I better remove it for now?

Reply:
yes